pkg/pgmodel/common/schema/schema.go

```diff
@@ -11,6 +11,8 @@ const (
 	Ext     = "_prom_ext"
 	Catalog = "_prom_catalog"
 
+	LockID = 9201 // Chosen randomly.
```

Comment:
We should not change this id
```suggestion
	LockID = 0x4D829C732AAFCEDE  // Chosen randomly.
```